superset-frontend/src/middleware/loggerMiddleware.js

```diff
@@ -116,7 +116,11 @@ const loggerMiddleware = store => next => action => {
     };
   }
 
-  if (eventData.target_id && dashboardLayout.present) {
+  if (
+    eventData.target_id &&
+    dashboardLayout.present &&
+    dashboardLayout.present[eventData.target_id]
```

Comment:
can we simplify? 
```
eventData.target_id &&
    dashboardLayout.present?.[eventData.target_id]
```

Reply:
good catch - should it be `dashboardLayout?.present?.[eventData.target_id]`